.travis.yml

```diff
@@ -52,3 +52,9 @@ deploy:
   on:
     tags: true
     condition: $DEPLOY_ARTIFACTS = 1
+
+notifications:
+  slack:
+    rooms:
+      - secure: "4NsPGDKKtiUSdqjl3bxUMlbw"
+    on_success: always
```

Comment:
I want it ping for every single push, so the pusher gets notified immediately if it fails. I've fixed the indent issue and added the on_failure line.